assets/airports/essa.json

```diff
@@ -0,0 +1,2865 @@
+{
+    "_comment": "Source airport info: https://www.aro.lfv.se/Editorial/View/7952/ES_AD_2_ESSA_en",
+    "airac": 2101,
+    "radio": {
+        "twr": "Arlanda Tower",
+        "app": "Stockholm Control",
+        "dep": "Stockholm Control"
+    },
+    "icao": "ESSA",
+    "iata": "ARN",
+    "magnetic_north": 6,
+    "ctr_radius": 150,
```

Comment:
This will likely have to be tuned after I use the airport a little and see how it feels.